src/tscpp/api/utils_internal.cc

```diff
@@ -141,6 +155,15 @@ void inline invokePluginForEvent(Plugin *plugin, TSHttpTxn ats_txn_handle, TSEve
   case TS_EVENT_HTTP_CACHE_LOOKUP_COMPLETE:
     plugin->handleReadCacheLookupComplete(transaction);
     break;
+  case TS_EVENT_HTTP_TXN_CLOSE:
+    if (plugin) {
```

Comment:
But doesn't this mean you could not be sure that your override of handleTxnClose() would actually get run?  Because the continuation that runs handleTransactionEvents() may run first.  I don't think we ever merged the proposed continuation priority changes (@duke8253 @SolidWallOfCode  ?).  I think thats what you'd need for this to be useful.

Reply:
No, the continuation will still run regardless of `handleTransactionEvents()`. That's how the stack trace above resulted.

Reply:
OK right.  But you may have other problems then.  You won't run the work-arounds in handleTransactionEvents(), that other features in the CPPAPI depend on.